README.adoc

```diff
@@ -0,0 +1,33 @@
+= CodeReady Containers - OpenShift 4 on your Laptop
```

Comment:
:+1: for converting this to AsciiDoc, I was about to open an issue for this myself. :)